### PR TITLE
Docs postgresql 9.1 merge

### PR DESCRIPTION
**Pull request by dyozie**
Greenplum doc updates to bring us in line with the 9.1 merge (https://github.com/greenplum-db/gpdb/commit/25a90396cd1c52d252a37986e12b63e8e037aa83)

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -92,6 +92,9 @@
               <xref href="#default_tablespace"/>
             </li>
             <li><xref href="#default_text_search_config" format="dita"/></li>
+            <li>
+              <xref href="#eefault_transction_deferrable"/>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: *ee*fault_transaction_deferrable ->*de*fault_transaction_deferrable?

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 2]
```diff
@@ -1659,6 +1662,44 @@
       </table>
     </body>
   </topic>
+  <topic id="default_transaction_deferrable">
+    <title>default_transaction_deferrable</title>
+    <body>
+      <note>Only read-only, serializable transactions can be deferred. Greenplum Database does not
+        support the <codeph>SERIALIZABLE</codeph> transaction isolation level, so setting
+          <codeph>default_transaction_deferrable</codeph> to on has no effect in Greenplum
+        Database.</note>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: if the default setting is off, and setting this to on has no effect, why do we include the text below and the table?

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_OPERATOR.xml [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+anyway.) </p></section><section id="section4"><title>Parameters</title><parml><plentry><pt><varname>name</varname></pt><pd>The name (optionally schema-qualified) of an existing operator. </pd></plentry><plentry><pt><varname>left_type</varname></pt><pd>The data type of the operator's left operand; write <codeph>NONE</codeph>
+if the operator has no left operand. </pd></plentry><plentry><pt><varname>right_type</varname></pt><pd>The data type of the operator's right operand; write <codeph>NONE</codeph>
+if the operator has no right operand. </pd></plentry><plentry><pt><varname>new_owner</varname></pt><pd>The new owner of the operator. </pd></plentry>
+        <plentry>
+          <pt><varname>new_schema</varname></pt>
+          <pd>The new schema for the opertor.</pd>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/sql_commands/ALTER_OPERATOR.xml]: opertor -> operator?

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_DOMAIN.xml [hunk 4]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE topic
   PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
-<topic id="topic1"><title id="bq20941">CREATE DOMAIN</title><body><p id="sql_command_desc">Defines a new domain.</p><section id="section2"><title>Synopsis</title><codeblock id="sql_command_synopsis">CREATE DOMAIN <varname>name</varname> [AS] <varname>data_type</varname> [DEFAULT <varname>expression</varname>] 
+<topic id="topic1"><title id="bq20941">CREATE DOMAIN</title><body><p id="sql_command_desc">Defines a new domain.</p><section id="section2"><title>Synopsis</title><codeblock id="sql_command_synopsis">CREATE DOMAIN <varname>name</varname> [AS] <varname>data_type</varname> [DEFAULT <varname>expression</varname>]
+       [ COLLATE <varname>collation</varname> ]
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_DOMAIN.xml]: remove space before COLLATE

File: gpdb-doc/dita/ref_guide/sql_commands/LOCK.xml [hunk 5]
```diff
@@ -35,14 +35,15 @@
...
-                before executing any <codeph>SELECT</codeph> or data modification statement. A
-                serializable transaction's view of data will be frozen when its first
-                    <codeph>SELECT</codeph> or data modification statement begins. A <codeph>LOCK
-                    TABLE</codeph> later in the transaction will still prevent concurrent writes —
-                but it won't ensure that what the transaction reads corresponds to the latest
-                committed values.</p>
+            <p>To achieve a similar effect when running a transaction at the <codeph>REPEATABLE
+                    READ</codeph> or <codeph>SERIALIZABLE</codeph> isolation level, you have to
+                execute the <codeph>LOCK TABLE</codeph> statement before executing any
+                    <codeph>SELECT</codeph> or data modification statement. A <codeph>REPEATBLE
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/LOCK.xml]: REPEATBLE > REPEATABLE

File: gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml [hunk 6]
```diff
@@ -74,27 +74,34 @@ with_query_name</varname> [ [AS] <varname>alias</varname> [( <varname>column_ali
...
         <li id="eg141787">If a window expression is specified (and optional <codeph>WINDOW</codeph>
           clause), the output is organized according to the positional (row) or value-based (range)
           window frame.</li>
-        <li id="eg141798"><codeph>DISTINCT</codeph> eliminates duplicate rows from the result.
-            <codeph>DISTINCT ON</codeph> eliminates rows that match on all the specified
-          expressions. <codeph>ALL</codeph> (the default) will return all candidate rows, including
-          duplicates.</li>
         <li id="eg138848">The actual output rows are computed using the <codeph>SELECT</codeph>
           output expressions for each selected row.</li>
+
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml]: remove blank line

File: gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml [hunk 7]
```diff
@@ -116,7 +123,14 @@ with_query_name</varname> [ [AS] <varname>alias</varname> [( <varname>column_ali
         <p>The optional <codeph>WITH</codeph> clause allows you to specify one or more subqueries
           that can be referenced by name in the primary query. The subqueries effectively act as
           temporary tables or views for the duration of the primary query. Each subquery can be a
-            <codeph>SELECT</codeph>, or <codeph>VALUES</codeph> command. </p><p>A
+            <codeph>SELECT</codeph>, <codeph>INSERT</codeph>, <codeph>UPDATE</codeph> or
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml]: oxford comma

File: gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml [hunk 8]
```diff
@@ -204,6 +229,32 @@ SELECT * FROM foo LIMIT 5;</codeblock></p><p>See
           coming from just that table. In these cases it is not possible to specify new names with
             <codeph>AS</codeph>; the output column names will be the same as the table columns'
           names.</p></sectiondiv>
+      <sectiondiv><b>The DISTINCT Clause</b>
+        <p>If <codeph>SELECT DISTINCT</codeph> is specified, all duplicate rows are removed from the
+          result set (one row is kept from each group of duplicates). <codeph>SELECT ALL</codeph>
+          specifies the opposite: all rows are kept; that is the default. </p>
+
```
Comment (litzell) [inline gpdb-doc/dita/ref_guide/sql_commands/SELECT.xml]: remove blank lines

**Pull request merged**